.github/workflows/benchmarks.yml

```diff
@@ -130,22 +130,24 @@ jobs:
         id: build
         run: |
           ./build_tools/github_actions/docker_run.sh \
-            --env "${PLATFORM^^}_ARCH=${ARCHITECTURE}" \
+            --env "TARGET_PLATFORM=${PLATFORM}" \
```

Comment:
I think we should probably prefix our environment variables with "IREE". That avoids name conflicts

Reply:
Done.